src/BuildingRegistry.Projections.Legacy/BuildingUnitDetailV2/BuildingUnitDetailV2Projections.cs

```diff
@@ -74,6 +73,16 @@ public BuildingUnitDetailV2Projections()
                 await context.BuildingUnitBuildingsV2.AddAsync(building, ct);
             });
 
+            When<Envelope<UnplannedBuildingWasRealizedAndMeasured>>(async (context, message, ct) =>
+            {
+                var building = new BuildingUnitBuildingItemV2(
```

Comment:
@ArneD why do we need to add this to building unit? 
Also do we need to add the status to realized?